docs/mesos/ManualInstallCalico.md

```diff
@@ -0,0 +1,63 @@
+# Manually Install the Calico Mesos Plugin
+This guide will walk you through manually downloading and installing the Calico-Mesos network plugin for your Mesos deployment. 
+
+The Calico-Mesos plugin must be installed with Calico's core services running on every Agent node in your cluster in order to take advantage of Calico networking. 
+
+*This guide assumes you already have Mesos + Net-Modules installed on your Agent(s).* 
+
+> To obtain the Calico files, you will need `wget` installed. Please install the tool if you haven't already done so.
+
+## Download the Calico Mesos Plugin
+
+The Calico-Mesos plugin is available for download under releases on the [calico-mesos repository](https://github.com/projectcalico/calico-mesos). We recommend installing this binary in the `/calico` directory.
```

Comment:
`...for download from the [calico-mesos repository releases](https://github.com/projectcalico/calico-mesos/releases/)`